action.yml

```diff
@@ -44,6 +44,10 @@ inputs:
     description: 'Optional state directory to use (if unset, memory state is used)'
     required: false
     default: ''
+  timeout:
```

Comment:
I'd rather not expand the API surface of the action if we really don't need to.  Users can already set the timeout on either the entire job or the step, so I'm not sure that having a configurable timeout is necessary here also.

Reply:
oh, I just noticed that the original issue this was fixing was specifically to add a configurable timeout.  I'm still curious if the step timeout does not also accomplish the same thing?